utility/mirrormanager2.spec

```diff
@@ -576,6 +643,9 @@ MM2_SKIP_NETWORK_TESTS=1 ./runtests.sh -v
 - umdl: various fixes found in production
   https://github.com/fedora-infra/mirrormanager2/pull/211
 
+* Wed Jul 26 2017 Fedora Release Engineering <releng@fedoraproject.org> - 0.8-2
+- Rebuilt for https://fedoraproject.org/wiki/Fedora_27_Mass_Rebuild
+
```

Comment:
Not sure we need to copy all of the changelog entries.

Reply:
This is a result of updating the upstream spec file with the downstream changes. From now on, packit should propose a PR for downstream from the updated spec file in upstream. In case I don't include all of these Changelog entries from downstream, they might be lost.

Reply:
From my point of view it does not make much sense to carry downstream commit messages in the upstream spec file.

I am undecided. I probably would just go with `%autochangelog` but that does not work on EPEL 7. I think. I would skip this change.

Reply:
Removing downstream commit messages.